experimental/devices/mfrc522/mfrc522.go

```diff
@@ -123,25 +123,30 @@ func NewSPI(spiPort spi.Port, resetPin gpio.PinOut, irqPin gpio.PinIn) (*Dev, er
 		antennaGain:      4,
 		stop:             make(chan struct{}, 1),
 	}
-	if err := dev.Init(); err != nil {
+	if err := dev.init(); err != nil {
 		return nil, err
 	}
 	return dev, nil
 }
 
 // Dev is an handle to an MFRC522 RFID reader.
 type Dev struct {
-	resetPin         gpio.PinOut
-	irqPin           gpio.PinIn
+	resetPin gpio.PinOut
+	irqPin   gpio.PinIn
+	spiDev   spi.Conn
+	stop     chan struct{}
+
+	oMu              sync.Mutex
```

Comment:
Please use the same mutex for both antennaGain and operationTimeout, there's no need to use separate mutex.